.github/workflows/auto-merge.yml

```
@@ -20,23 +18,38 @@ concurrency:
 
 jobs:
   dependabot:
+    name: Dependabot PR auto-merge
     runs-on: ubuntu-latest
     if: ${{ github.actor == 'dependabot[bot]' }}
     steps:
-      - name: Dependabot metadata
-        id: metadata
-        uses: dependabot/fetch-metadata@v1.3.3
-        with:
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-      - name: Enable auto-merge for Dependabot PRs
-        if: ${{ github.event.pull_request.auto_merge == null && ( steps.metadata.outputs.update-type == 'version-update:semver-patch' || steps.metadata.outputs.update-type == 'version-update:semver-minor' ) }}
-        run: gh pr merge --auto --merge "$PR_URL"
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Approve and merge the PR
+        if: |
+          contains( github.event.workflow_run.head_commit.message, 'version-update:semver-minor' ) ||
+          contains( github.event.workflow_run.head_commit.message, 'version-update:semver-patch' )
+        run: |
+          gh pr review "$PR_NUMBER" --approve --body "LGTM!"
+          gh pr merge "$PR_NUMBER" --auto --merge --delete-branch
         env:
-          PR_URL: ${{github.event.pull_request.html_url}}
+          PR_NUMBER: ${{github.event.workflow_run.pull_requests[0].number}}
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
-      - name: Approve PR
-        run: gh pr review --approve "$PR_URL"
-        if: ${{ steps.metadata.outputs.update-type == 'version-update:semver-patch' || steps.metadata.outputs.update-type == 'version-update:semver-minor' }}
+
+  gutenberg-packages:
+    name: Gutenberg packages update PR auto-merge
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    if: ${{ github.actor == 'user-with-pat' }}
```

@westonruter Once you add `personal-access-token` in GHA secrets please replace  `user-with-pat` with the username of the user whose GH PAT has been added to trigger workflows.